src/util/system.cpp

```diff
@@ -1200,25 +1200,30 @@ UniValue RunCommandParseJSON(const std::string& str_command, const std::string&
 
     if (str_command.empty()) return UniValue::VNULL;
 
-    bp::child c(
+    // This is here because boost::process quotes the argument given to
+    // system when used in combination with boost::process::shell.
+    // https://github.com/boostorg/process/issues/95
```

Comment:
Does bp::child need this workaround too? If not, that seems like a reason to avoid bp::system...